cli_tools/gce_windows_upgrade/upgrade_script_2008r2_to_2012r2.ps1

```diff
@@ -0,0 +1,65 @@
+#  Copyright 2020 Google Inc. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+$script:install_media_drive = ''
+
+try {
+  Write-Host 'GCEMetadataScripts: Beginning upgrade startup script.'
+
+  # cleanup garbages from a failed upgrade to unblock a new upgrade
```

Comment:
Instead of 'garbages', please describe, precisely what you're expecting to delete.

Also, for comments, here and elsewhere, please use a full sentence, with a capital and a period :)